### DISCUSSION TITLE
Erro no publish do pacote MoneyExtensions

### DISCUSSION THREAD

**Discussion by LeonardoGuarilha** [balta-io/forum#80] [Tira Dúvidas] [answered]
CURSO: Fundamentos do Azure, Git, GitHub e DevOps

Boa tarde,
Segui todas as aulas até a parte de fazer o push do código para o github, mas na hora de rodar a action é apresentado o seguinte erro
![image](https://user-images.githubusercontent.com/39388688/200073574-2906adf0-1e34-445f-b6a1-e990fe078a8d.png)

Meu arquivo main.yml está da seguinte forma
![image](https://user-images.githubusercontent.com/39388688/200073644-16102f5e-8602-42b3-9073-a4bb8722d09c.png)

Tenho também a secret criada no nuget
![image](https://user-images.githubusercontent.com/39388688/200073891-7404cebd-631f-49a8-b3f4-56c475d45bc4.png)

E por fim no repositório do github
![image](https://user-images.githubusercontent.com/39388688/200074002-ce3656ca-79ae-46fa-98fc-724162747933.png)

Na secret do git, eu só coloquei o nome e colei o valor da secret gerada pelo nuget

PS: Consegui descobrir o que era o problema. O problema era o nome do pacote, depois de colocar um nome único funcionou normalmente.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @LeonardoGuarilha como vai?
Isso aí, seu pacote estava com mesmo nome do meu! 
Abração!!!
